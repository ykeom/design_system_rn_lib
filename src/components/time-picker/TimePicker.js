import React, { Component } from 'react';
import { View, StyleSheet, Text } from 'react-native';

const styles = StyleSheet.create({
  root: {},
});

class TimePicker extends Component {
  render() {
    return (
      <View style={styles.root}>
        <Text>time picker</Text>
      </View>
    );
  }
}

export default TimePicker;
