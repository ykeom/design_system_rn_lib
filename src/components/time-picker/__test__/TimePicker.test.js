import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import TimePicker from '../TimePicker';

describe('TimePicker', () => {
  it('should render', () => {
    const component = shallow(<TimePicker />);

    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});
