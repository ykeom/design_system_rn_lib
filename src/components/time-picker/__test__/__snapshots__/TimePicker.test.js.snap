// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimePicker should render 1`] = `
<View
  style={Object {}}
>
  <Text>
    time picker
  </Text>
</View>
`;
